lib/manager/bundler/extract.ts

```diff
@@ -116,7 +116,7 @@ export async function extractPackageFile(
           sourceLine = lines[lineNumber];
           // istanbul ignore if
           if (sourceLine === null || sourceLine === undefined) {
-            logger.error({ content, fileName }, 'Undefined sourceLine');
+            logger.info({ content, fileName }, 'Undefined sourceLine');
```

Comment:
I picked info based on the guidelines in https://github.com/renovatebot/renovate/blob/8e97958dd17b3ec92926af10fc012bea66308d26/docs/development/design-decisions.md#logging-and-error-levels